src/CLI/Command.php

```
@@ -0,0 +1,176 @@
+	 * ---
+	 * default: false
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *    wp product-feed generate
+	 *    wp product-feed generate --timeout=200
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Associative arguments.
+	 */
+	public function generate( $args, $assoc_args ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)